src/services/github_service.ts

```diff
@@ -0,0 +1,56 @@
+interface nestModule {
+  name: string;
+  normalizedName: string;
+  owner: string;
+  description: string;
+  repository: string;
+  latestVersion: string;
+  latestStableVersion: string;
+  packageUploadNames: string[];
+  locked: boolean | null;
+  malicious: boolean | null;
+  unlisted: boolean;
+  updatedAt: string;
+  createdAt: string;
+}
+
+export default class GitHubService {
+  /**
+   * Fetches the latest release of a a github repository
```

Comment:
fetches ... of a a a a a a a a a a a a a a a

Reply:
lol ;)

Reply:
https://github.com/drashland/dmm/pull/91/commits/0bde1b214e9e42bbd7be9ad86e5303b3be1d2a2a